packages/tx/README.md

```diff
@@ -441,42 +441,67 @@ To sign a tx with a hardware or external wallet use `tx.getMessageToSign()` to r
 
 A legacy transaction will return a Buffer list of the values, and a Typed Transaction ([EIP-2718](https://eips.ethereum.org/EIPS/eip-2718)) will return the serialized output.
```

Comment:
This is not edited by this PR but the note of "Buffer" means that these edits are in very historic parts of the README and that those are also incorrect (we now use `Uint8Array` instead of `Buffer`)